lms/djangoapps/ccx/tests/test_field_override_performance.py

```diff
@@ -214,24 +214,24 @@ class TestFieldOverrideMongoPerformance(FieldOverridePerformanceTestCase):
 
     TEST_DATA = {
         # (providers, course_width, enable_ccx, view_as_ccx): # of sql queries, # of mongo queries, # of xblocks
-        ('no_overrides', 1, True, False): (23, 7, 14),
-        ('no_overrides', 2, True, False): (68, 7, 85),
-        ('no_overrides', 3, True, False): (263, 7, 336),
-        ('ccx', 1, True, False): (23, 7, 14),
-        ('ccx', 2, True, False): (68, 7, 85),
-        ('ccx', 3, True, False): (263, 7, 336),
-        ('ccx', 1, True, True): (23, 7, 14),
-        ('ccx', 2, True, True): (68, 7, 85),
-        ('ccx', 3, True, True): (263, 7, 336),
-        ('no_overrides', 1, False, False): (23, 7, 14),
-        ('no_overrides', 2, False, False): (68, 7, 85),
-        ('no_overrides', 3, False, False): (263, 7, 336),
-        ('ccx', 1, False, False): (23, 7, 14),
-        ('ccx', 2, False, False): (68, 7, 85),
-        ('ccx', 3, False, False): (263, 7, 336),
-        ('ccx', 1, False, True): (23, 7, 14),
-        ('ccx', 2, False, True): (68, 7, 85),
-        ('ccx', 3, False, True): (263, 7, 336),
+        ('no_overrides', 1, True, False): (24, 7, 14),
```

Comment:
Speaking of SQL calls, I think it'd be worthwhile to have a test case which verifies that the cache does what it's supposed to, though I know we've had issues with that type of test case failing on jenkins before.